client/containers/Pub/PubSyncManager.tsx

```diff
@@ -160,8 +159,11 @@ class PubSyncManager extends React.Component<Props, State> {
 	componentDidMount() {
 		const rootKey = `pub-${this.props.pubData.id}`;
 		const branchKey = `branch-${this.props.pubData.activeBranch.id}`;
-		// @ts-expect-error ts-migrate(2569) FIXME: Type 'Reference[] | null' is not an array type or ... Remove this comment to see the full error message
-		initFirebase(rootKey, this.props.pubData.firebaseToken).then(([rootRef, connectionRef]) => {
+		initFirebase(rootKey, this.props.pubData.firebaseToken).then((firebaseRefs) => {
+			if (!firebaseRefs) {
+				return;
+			}
+			const [rootRef, connectionRef] = firebaseRefs;
```

Comment:
I was able to remove a TS error by inserting an early return here. I didn't do too much digging into the types to figure out why `firebaseRefs` would be null or undefined. @idreyn is this okay for now or should we throw an error instead of failing silently?

Reply:
`initFirebase` will return `null` if there's an authentication error. My guess is that in either case the failure is equivalently silent to the user, and they'll just see "Connecting..." in the status bar.